### PR TITLE
Add design to pagetree

### PR DESCRIPTION
**Pull request by FinalAngel**
please ignore `cms/static/cms/js/libs/jstree/jstree.grid.js` as its a library, will be minified in the cleanup branch.

image

**Tooltips**

image

image

**Copy & Paste**:

image

image

**Loading & Errors**

image

image

image

image

**Positioning**
image

### CODE REVIEW

File: cms/static/cms/js/libs/jstree/themes/django-cms/style.scss [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    .jstree-anchor {
+        box-sizing: border-box;
+        display: block;
+        cursor: move;
+        outline: none;
+        color: $gray;
+        // we need to add one pixel in to negate the negative margin
+        // value to get the borders to work visually
+        height: $pagetree-cell-height + 1;
+        padding: 9px 25px;
```
Comment (vxsx) [inline cms/static/cms/js/libs/jstree/themes/django-cms/style.scss]: would love to see these in variables

File: cms/static/cms/sass/components/_pagetree.scss [hunk 2]
```diff
@@ -2,10 +2,15 @@
 // PAGETREE
 
 $pagetree-icon-size: 18px;
-$pagetree-icon-offset: 4px;
-$pagetree-lang-size: 14px;
+$pagetree-icon-offset: 5px;
```
Comment (vxsx) [inline cms/static/cms/sass/components/_pagetree.scss]: maybe it's time to move them to cms settings?

File: cms/static/cms/js/libs/jstree/themes/django-cms/style.scss [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+        top: 4px;
+        color: $color-success;
+        @extend %icon;
+        @include icon(check);
+    }
+}
+
+.cms-tree-item-helpers a {
+    position: relative;
+    top: 2px;
```
Comment (vxsx) [inline cms/static/cms/js/libs/jstree/themes/django-cms/style.scss]: can it be done with having a height and positioning the icon in the center of the height instead?

**Comment by vxsx** [comment]
overall LGTM, but i'd prefer more variables, especially for icon positioning, heights, paddings etc

File: cms/static/cms/js/libs/jstree/themes/django-cms/style.scss (hunk 3 above)
Comment (FinalAngel) [inline cms/static/cms/js/libs/jstree/themes/django-cms/style.scss]: sadly not in this case

**Comment by vxsx** [comment]
:+1:

**Pull request merged**